cypress/locators/locators.js

```diff
@@ -6,6 +6,7 @@ export const CHARACTER_LIMIT = 'input[name="characterLimit"]';
 export const FORM = '#storybook-panel-root';
 export const TAB_LIST = '[role="tablist"]';
 export const CLOSE_ICON_BUTTON = 'button[data-element="close"]';
+export const INPUT_TAG = 'input';
```

Comment:
I would rather add `data-element="asterisk"` and search for it in cypress instead of searching the `input`

Reply:
We cannot use that in css.